manifests/pv/pvc.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: katib-volume
+  namespace: katib
+  labels:
+    app: katib
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: ""
+  selector:
+    matchLabels:
```

Comment:
no need for that if users have working storageclass. By the way, minikube comes with storageclass by default so it's unnecessary.